nservicebus/sqlserver/operations-scripting.md

```diff
@@ -90,3 +90,44 @@ snippet:sqlserver-return-to-source-queue
 ### Using the retry helper methods
 
 snippet:sqlserver-return-to-source-queue-usage
+
+
+### How to shift SqlTransport audit log to long term storage
+
+There are several ways to achieve this. If you have access to a Database Administrator talk to them about techniques like [Table Partitioning](https://technet.microsoft.com/en-us/library/ms188730.aspx) and [Snapshot Replication](https://technet.microsoft.com/en-us/library/ms151832.aspx).
+
+If you're just looking for some simple techniques to clear down your audit queue try this
+
+In SQL Management Studio create an audit_archive table with this SQL script.
+
+```
```

Comment:
Just to clarify what @weralabaj pointed out. We don't think that changes to the format are necessary for v3. There is however a plan to switch to stored procedures interface in the future but I don't think we are in position to make any good estimations on when that will happen.